R.swift/Generators/StringsGenerator.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  StringsGenerator.swift
+//  R.swift
+//
+//  Created by Nolan Warner on 2016/02/23.
+//  Copyright © 2016 Nolan Warner. All rights reserved.
+//
+
+import Foundation
+
+struct StringsGenerator: Generator {
+  let externalStruct: Struct?
+  let internalStruct: Struct? = nil
+
+  init(localizableStrings: [LocalizableStrings]) {
+
+    let localized = localizableStrings.groupBy { $0.filename }
```

Comment:
Why do we first `groupBy` on filename and then again `groupBySwiftNames`, this `localized` variable seems to be never used again. So maybe this can be done all at once?